client/src/main/scala/drt/client/components/DateSelector.scala

```diff
@@ -23,7 +22,7 @@ object DateSelector {
 
   val today: SDateLike = SDate.now()
 
-  def daysInMonth(month: Int, year: Int): Int = new Date(year, month, 0).getDate()
+  def daysInMonth(month: Int, year: Int): Int = new Date(year, month, 0).getDate().toInt
```

Comment:
same toInt may be redundant

Reply:
getDate is actually a JavaScript function which returns a JS Number type, which is actually a double. This is new type safety that we are getting from the newer version of ScalaJS.